File: modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/services/checker/trait/CheckerServiceOnSecondaryFileUpload.scala
package de.thm.ii.fbs.services.checker.`trait`

import de.thm.ii.fbs.model.CheckrunnerConfiguration
import de.thm.ii.fbs.model.task.Task

trait CheckerServiceOnSecondaryFileUpload {
  def onCheckerSecondaryFileUpload(cid: Int, task: Task, checkerConfiguration: CheckrunnerConfiguration): Unit
}

File: modules/fbs-core/api/src/main/kotlin/de/thm/ii/fbs/services/v2/checker/trait/CheckerServiceOnSecondaryFileUpload.kt
package de.thm.ii.fbs.services.v2.checker.trait

import de.thm.ii.fbs.model.v2.CheckrunnerConfiguration
import de.thm.ii.fbs.model.v2.Task

interface CheckerServiceOnSecondaryFileUpload {
    fun onCheckerSecondaryFileUpload(cid: Int, task: Task, checkerConfiguration: CheckrunnerConfiguration)
}